.github/workflows/test.yml

```diff
@@ -3,8 +3,7 @@ name: test
 on:
   push:
     branches:
-      - master
-      - 'release/**'
+      - '*'
```

Comment:
I do not think changes to the tests should be mixed into PRs that fix bugs or implement new features, but should be in their own PR.

I'm not sure why you would want to run tests on each push to any branch. I worry that this could use up some Github quota, so other tests may be queued later, instead of being executed right away. Not sure if that is a valid concern though.